replicatelogs.go

```diff
@@ -119,7 +130,7 @@ By default transient errors are re-tried without limit, and if the error is 429,
 				return err
 			}
 
-			changes, err := readTenantMassifChanges(cCtx)
+			changes, err := readTenantMassifChanges(context.TODO(), cCtx, cmd)
```

Comment:
guessing we don't really have a better context here? maybe short comment to explain what the plan for this one is?